Roys/app/src/main/java/com/dwg_karrier/roys/ListViewAdapter.java

```diff
@@ -0,0 +1,53 @@
+package com.dwg_karrier.roys;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+
+/**
+ * Created by userpc on 2017-01-16.
+ */
+
+public class ListViewAdapter extends BaseAdapter {
+
+  private LayoutInflater inflater;
+  private ArrayList<item> data;
+  private int layout;
+
+  public ListViewAdapter(Context contest, int layout, ArrayList<item> data) {
+    this.inflater = (LayoutInflater)contest.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+    this.data  = data;
+    this.layout = layout;
+  }
+  @Override
+  public int getCount()
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace
Please write the starting { with the same line.